chart/localdb/charts/dbdump/values.yaml

```diff
@@ -0,0 +1 @@
+global:
```

Comment:
The pattern is to use global for the overrides that are in the top-level chart's values.yaml.

In this case that is the localdb values.yaml, they are global - valid for all subcharts.

I propose to remove it and leave the file empty and once we need overrides here we will see what the context will be and will name it accordingly.